Fix UI bug - remark overflowing

jeffreytjs opened the pull request:
- Resolved remarks overflowing instead of truncating in the GUI.
- Instead of truncating, remarks and names now set to wrap
- Improved compatibility with DarkTheme

jeffreytjs committed:
Fix Remark overflow bug

- Replace Flowpane with Label and adjust styling using css

jeffreytjs committed:
Fix Label being truncated

- Allow wrapping

zhengweii commented on src/main/resources/view/LightTheme.css:
```diff
@@ -63,12 +63,12 @@
 
-#remark {
+.remark {
     -fx-hgap: 7;
     -fx-vgap: 3;
 }
 
-#remark .label {
+.remark {
     -fx-background-color: #6FB6ED; /*light blue for tags*/
```
Hmm would it be possible to combine these two?

jeffreytjs replied:
Yea definitely!

jeffreytjs committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2103-W14-4/tp into fix-GUI-bugs

jeffreytjs committed:
Simplify code and ensure compatibility with DarkTheme

- Correct colour combination for DarkTheme, product font was white against white background

jeffreytjs committed:
Compatibility with DarkTheme

- Correct colour combination for DarkTheme, product font was white against white background

qlchan24 reviewed (approved):
LGTM
